tests/test_modeling_common.py

```diff
@@ -4999,82 +4718,6 @@ def test_custom_4d_attention_mask(self):
             normalized_1 = F.softmax(out_shared_prefix_last_tokens)
             torch.testing.assert_close(normalized_0, normalized_1, rtol=1e-3, atol=1e-4)
 
-    def test_static_cache_matches_dynamic(self):
```

Comment:
Merged the checks in this test [check that the outputs are the same] with the checks in `test_generate_with_static_cache` [check that the shapes are as expected]